cmd/kpod/create.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+)
+
+// TODO: Missing flags from docker-create - particularly --security-opt or equiv and resource limit flags
+// TODO stop using Kubernetes API structs here - replace with our own versions (remove pb import entirely)
+// TODO Add missing flags from docker-create
+// TODO parse flags into a createConfig, and parse createConfig into an OCI runtime spec
+
+// TODO These temporary values should be replaced with sane defaults
+const (
+	defaultHostname     = "kpod-launch"
+	defaultCgroupParent = "/kpod-launch"
+)
+
+type createConfig struct {
+	image            string
+	command          string
+	args             []string
+	pod              string
+	privileged       bool
+	rm               bool
+	hostNet          bool
+	hostPID          bool
+	hostIPC          bool
+	name             string
+	labels           map[string]string
+	workDir          string
+	env              map[string]string
+	detach           bool
+	stdin            bool
+	tty              bool
+	devices          []*pb.Device
+	mounts           []*pb.Mount
+	capAdd           []string
+	capDrop          []string
+	dnsServers       []string
+	dnsSearch        []string
+	dnsOpt           []string
+	ports            []*pb.PortMapping
+	hostname         string
+	cgroupParent     string
+	sysctl           string
+	user             int64
+	additionalGroups []int64
+	readOnlyRootfs   bool
+}
+
+var createFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "pod",
+		Usage: "Run container in an existing pod",
+	},
+	cli.BoolFlag{
+		Name:  "privileged",
```

Comment:
alphabetize

Reply:
The flags section should be in a separate go file so that it can be added to create and run.